llmv_routes.py

```diff
@@ -39,6 +41,12 @@ def llm_verification_index():
     @authed_only
     def generate_for_challenge():
         """Add a route to CTFd for generating text from a prompt."""
+        log.error("Log name %s, %s", log.name, log.level)
+        log.info("Received test generation request but his logger is not working for some reason")
+        log.error("NEW Received test generation request. Testing error level logging too")
```

Comment:
For some reason info level logs don't print to console which is the main issue
![image](https://github.com/aivillage/llm_verification/assets/7213793/d7bccf17-82ed-447d-bca7-df6b20b9fd8a)

Reply:
Something funny is going on where the llmv_routes logging is going to the wrong logger

![image](https://github.com/aivillage/llm_verification/assets/7213793/2b2b73ad-c4de-4ef9-809b-8ef559cddabc)

Reply:
Maybe this. I'll figure it out tomorrow

https://stackoverflow.com/questions/27335686/adding-blueprints-to-flask-seems-to-break-logging